zeebe/atomix/cluster/src/main/java/io/atomix/cluster/messaging/MessagingConfig.java

```diff
@@ -33,6 +33,8 @@ public class MessagingConfig implements Config {
   private File certificateChain;
   private File privateKey;
   private CompressionAlgorithm compressionAlgorithm = CompressionAlgorithm.NONE;
+  private File pkcs12;
+  private String pkcs12Password;
```

Comment:
This can't be final unless it's assigned in the constructor and I think the pattern is a pseudo builder where everything is configured with methods and not constructor arguments.